src/com/raghav/javabasics/day4/PrintDuplicateCharInString.java

```diff
@@ -0,0 +1,46 @@
+package com.raghav.javabasics.day4;
+//Given a string, write a program to print the duplicate characters in the String.
+
+public class PrintDuplicateCharInString {
+    boolean abc= false;
+    public static void main(String[] args) {
+        PrintDuplicateCharInString result = new PrintDuplicateCharInString();
+        result.findDuplicates("program to print the duplicate characters in the String.");
```

Comment:
Output:
[p][r][o][g][a][ ][t][i][n][h][e][c][ ]Array is Empty/Null
Array is Empty/Null
[a][b][ ]